test/e2e/cypress/fixtures/sap-systems-overview/sap_systems_overview.feature

```diff
@@ -0,0 +1,68 @@
+Feature: SAP Systems Overview
+    This is where the user has an overview of the SAP Systems and their attached HANA databases.
+
+    Background:
+        Given an healthy SAP deployment consisting of 27 hosts, 1 hana cluster, 1 netweaver cluster
+        Given 3 Application SAP Systems with sids 'NWP', 'NWQ', 'NWD'
+        Given 1 HANA database for each SAP System 'HDP', 'HDQ', 'HDD'
+        And a Trento installation on this Cluster
+
+    Scenario: Registered SAP Systems should be available in the overview
+        When I navigate to the SAP Systems overview page
+        Then the discovered SID ar the expected ones
```

Comment:
typo: 
`Then the discovered SID ar the expected ones`
`Then the discovered SID are the expected ones`

Reply:
Damn!!
Time to add a syntax checker. 
I will create a tech-debt ticket in our backlog